docs/best_practices.md

```diff
@@ -0,0 +1,152 @@
+command will allow you to create multiple Yubikeys with the Root key loaded.
+
+Additionally, you may choose whether or not to re-use a Root key for multiple repositories.
+Notary will automatically use the first Root key it finds available, or generate a new Root
+key if none is found. By ensuring only the Root key you want to create the repository with
+is available, i.e. by attaching the appropriate Yubikey, Notary will use this key as the
+root key for a new repository.
+
+## Key Rotations
+
+If you follow the appropriate security measures, you should not need to rotate your Root
+key until the signing mechanism in use is obsoleted. We currently default to ECDSA 
+with the P256 curve and do not expect it to be obsoleted for many years. Currently though
+the Root key is published as a self signed x509 certificate with an expiry set 10 years
+in the future. Therefore, you will need to rotate your root key within this time frame.
+While the root key shouldn't need to be rotated often, it is still beneficial to periodically
+require an update to improved or recommended cipher suites. We chose a 10 year expiry as the
+default in order to enforce this requirement.
+
+It may however
+be desirable to rotate other keys at more frequent intervals due to their increased exposure
+to potential compromise. We have no specific recommendations on the period length.
+
+## Expiration Prevention
+
+Keys held online by the Notary signer will be used to re-sign their associated roles
+if the current metadata expires. The current default metadata expiry times, which will
+be reduced as people become familiar with the management of Notary repositories, are:
+
+* Root - 10 years
+* Targets and Delegations - 3 years
+* Snapshot - 3 years
+* Timestamp - 14 days
+
+If you are publishing content that should not be expired after 3 years, you will need to
+ensure you have a mechanism in place to re-sign your content before it expires. If the
+Targets role expires, no content in the Trusted Collection will be considered valid by
+client. If however a delegation expires, only the items in the subtree rooted at that
+delegation will be invalidated.
+
+If your Root file expires, the repository as a whole is no longer trusted and all clients
+will fail when interacting with your repository. The Notary client, and the Docker Content
+Trust integration, will attempt to re-sign the Root file if it is within 6 months of expiring
+when a publish is performed.
+
+All roles other than the Timestamp will cause warnings to be printed to the logs (or
+the terminal in the case of the Notary or Docker CLIs) whenever the Notary repository
+is retrieved and a role is found to be within 6 months of expiring. This is to notify
+both consumers, and administrators that content they were trusting may become invalid
+due to inactivity.
+
+## Bootstrapping Trust
+
+By default Notary uses a mechanism we have termed TOFUS (Trust On First Use over SSL).
```

Comment:
nit: in other docs (client config) and comments we term TOFUS as `TOFU over HTTPS`.  We should just pick one for consistency